### PR TITLE
Allow control bar to be moved to the right side

### PR DESCRIPTION
**Pull request by CendioOssman**
In case the remote session has something on the left side that makes the control bar's placement there inconvenient.

### CODE REVIEW

File: app/ui.js [hunk 1]
```diff
@@ -576,11 +576,35 @@ var UI;
...
+            UI.controlbarDrag = true;
+        },
+
         dragControlbarHandle: function (e) {
             if (!UI.controlbarGrabbed) return;
 
             var ptr = Util.getPointerEvent(e);
 
+            var anchor = document.getElementById('noVNC_control_bar_anchor');
+            if (ptr.clientX < (window.innerWidth * 0.1)) {
```
Comment (samhed) [inline app/ui.js]: I'm thinking that we should probably try to use a physical width here in the future. However, not a big deal

**Comment by CendioOssman** [comment]
> The only drawback is that I feel that most noVNC users won't know that this feature exists. Is there any way we can display a graphical hint that in some way shows that the control bar can be moved to the other side? I'm absolutely not an expert, but some sort of shadow or arrow on the other side perhaps?

No idea right now. We'd want something that's clear and obvious, but at the same time not annoy users who already know they can do this.

File: app/ui.js (hunk 1 above)
Comment (CendioOssman) [inline app/ui.js]: Maybe. The advantage of a percentage is there is no risk of overlap. Seems to work well in practice right now, but we can always tweak it in the future.

**Comment by samhed** [comment]
> No idea right now. We'd want something that's clear and obvious, but at the same time not annoy users who already know they can do this.

Yeah.. I don't see it as a reason not to merge

**Pull request merged**